airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -63,24 +63,25 @@ export const JobItem: React.FC<JobItemProps> = ({ shortInfo, job }) => {
   };
 
   useEffectOnce(() => {
-    if (linkedJobId) {
-      scrollAnchor.current?.scrollIntoView({
-        behavior: "smooth",
-        block: "start",
-      });
+    if (linkedJobId === String(getJobId(job))) {
```

Comment:
Alternatively, I make this a negative check and return, then the rest of the code that returns the function.